src/connector/src/parser/unified/debezium.rs

```diff
@@ -165,13 +165,29 @@ pub fn parse_schema_change(
 ) -> AccessResult<SchemaChangeEnvelope> {
     let mut schema_changes = vec![];
 
-    let upstream_ddl = accessor
+    let upstream_ddl: String = accessor
         .access(&[UPSTREAM_DDL], &DataType::Varchar)?
         .to_owned_datum()
         .unwrap()
         .as_utf8()
         .to_string();
 
+    // Currently only accept ADD COLUMN and DROP COLUMN,
+    // and we assumes each schema change message only contains one DDL statement.
+    let allowed_ddl = ["ADD COLUMN", "DROP COLUMN"];
+    let upper_upstream_ddl = upstream_ddl.to_uppercase();
+    let is_allowed = allowed_ddl
+        .iter()
+        .any(|&allowed_ddl| upper_upstream_ddl.contains(allowed_ddl));
```

Comment:
This looks fragile. Shall we instead compare the previous and current schema to check whether one is the subset of the other?

Reply:
> compare the previous and current schema to check whether one is the subset of the other?

Not quite get it. Here the goal is to skip upstream messages generated by unsupported `ALTER` statements.

Reply:
Sounds ok to me, based on we only support `ADD` and `DROP`.